### PR TITLE
bluez 5.72 (new formula)

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

needed for #164059 for linux build.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  ./configure: line 14888: syntax error near unexpected token `GLIB,'
  ./configure: line 14888: `PKG_CHECK_MODULES(GLIB, glib-2.0 >= 2.28, dummy=yes,'
```

did not see in the linux runner.

File: Formula/b/bluez.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+class Bluez < Formula
+  desc "Bluetooth protocol stack for Linux"
+  homepage "https://github.com/bluez/bluez"
+  url "https://github.com/bluez/bluez/archive/refs/tags/5.72.tar.gz"
+  sha256 "0efb27303646b482518184c19ced08457f130b4ab69e3c952b9a448f986922ff"
```
Comment (cho-m) [inline Formula/b/bluez.rb:5]: Maybe use the kernel.org tarball: https://mirrors.edge.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz 
That should have `configure` script already generated.

**Comment by chenrui333** [comment]
actually missed the pkg-config build dep earlier 😅

File: Formula/b/bluez.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class Bluez < Formula
+  desc "Bluetooth protocol stack for Linux"
+  homepage "https://github.com/bluez/bluez"
+  url "https://mirrors.edge.kernel.org/pub/linux/bluetooth/bluez-5.72.tar.xz"
+  sha256 "499d7fa345a996c1bb650f5c6749e1d929111fa6ece0be0e98687fee6124536e"
+  license "GPL-2.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
```
Comment (p-linnane) [inline Formula/b/bluez.rb:10]: Do we still need these given the tarball switch?

Comment (chenrui333) [reply Formula/b/bluez.rb:10]: yeah, that is right, let me remove them.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**